### PR TITLE
Update boolean doc2

### PR DESCRIPTION
**Pull request by jsmcibm**
# Pull Request      

## What does this PR do?

updates to documentation of the boolean system to reflect the recently uploaded leaderboard models,
and deletion of files that were moved from boolqa to text_classification

```Closes #(issue)```       

Notes:
- Replace `(issue)` **above** ↑↑↑ with the issue this PR closes to automatically link the two.
This must be done when the PR is created.
- Add multiple `Closes #(issue)` as needed.
- If this PR is work towards but does not close an issue, simply tag the issue without mentioning `Closes`.

## Description

**Describe** the changes proposed by this PR below to give the reviewer context below ↓↓↓

```(description)```     

### Request Review

Be sure to **request a review** from one or more reviewers (unless the PR is to an unprotected branch).

## Versioning

When opening a PR to make changes to PrimeQA (i.e. `primeqa/`) master, be sure to increment the version following
[semantic versioning](https://semver.org/).  The VERSION is stored [here](https://github.com/primeqa/primeqa/blob/main/VERSION) 
and is incremented using `bump2version {patch,minor,major}` as described in the (development guide documentation)[development.html].

 - [ ] Have you updated the VERSION?
 - [ ] Or does this PR not change the `primeqa` package or was not into master?

After pulling in changes from master to an existing PR, ensure the VERSION is updated appropriately.
This may require bumping the version again if it has been previously bumped.

If you're not quite ready yet to post a PR for review, feel free to open a [draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/).

### After Merging
If merging into master and VERSION was updated, after this PR is merged:
- [ ] [Create a release](https://docs.github.com/en/github/administering-a-repository/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release) from the master with version equal to [VERSION](https://github.com/primeqa/primeqa/blob/main/VERSION)
- [ ] Not merging to master or VERSION not updated

### Checklist

Review the following and mark as completed:
- [ ] [Tag an issue or issues](#what-does-this-pr-do) this PR addresses.
- [ ] Added [description](#description) of changes proposed.
- [ ] Review requested as [appropriate](#request-review).
- [ ] Version bumped as [appropriate](#Versioning).
- [ ] New classes, methods, and functions documented.
- [ ] Documentation for modified code is updated.
- [ ] Built documentation to confirm it renders as expected (see [here](development.md)).
- [ ] Code cleaned up and commented out code removed.
- [ ] Tests added to ensure all functionalities tested at >= 60% unit test coverage (see [here](development.md)).
- [ ] Code cleaned up and commented out code removed.
- [ ] Release created [as needed](#after-merging) after merging.

### CODE REVIEW

File: primeqa/boolqa/README.md [hunk 1]
```diff
@@ -63,24 +63,24 @@ The configuration file contains the parameters for each of the post-MRC steps
         "id_key": "example_id",
         "sentence1_key": "question",
         "sentence2_key": null,
-        "label_list": ["short_answer","boolean"],
+        "label_list": ["boolean", "other"],
         "output_label_prefix": "question_type",
         "overwrite_cache": true,
         "use_auth_token": true,
-        "model_name_or_path": "PrimeQA/tydiqa-boolean-question-classifier"
+        "model_name_or_path": "PrimeQA/tydi-tydi_boolean_question_classifier-xlmr_large-20221117"
```
Comment (seirasto) [inline primeqa/boolqa/README.md:70]: Should the model name be changed to just have tydi in it once?

File: primeqa/boolqa/README.md [hunk 2]
```diff
@@ -131,28 +133,33 @@ a `yes` or `no` answer to question.  Both question and span are passed through t
...
+    --test_file ${BASE}/qtc/predictions.json \
+    --output_dir ${BASE}/evc \
     --use_auth_token
 ```
 
 ## Score normalization
 
 Span scores may have different dynamic ranges according as whether the question is boolean or short_anwer. Normalize them uniformally to `[0,1]`.
 and output a file suitable for the TyDiQA evaluation script.
+Warning: Because the score normalizer was developed for a leaderboard submission with hidden test set,
```
Comment (seirasto) [inline primeqa/boolqa/README.md:154]:
```suggestion
Warning: The score normalizer was developed for a leaderboard submission with a hidden test set.

```

File: primeqa/boolqa/README.md [hunk 3]
```diff
@@ -131,28 +133,33 @@ a `yes` or `no` answer to question.  Both question and span are passed through t
...
+the score normalizer component was trained on system output on half of the dev set.  As a result the full dev set results are contaminated.  The fair evalutation of the submission is on the leaderboard itself.
```
Comment (seirasto) [inline primeqa/boolqa/README.md:155]:
```suggestion
Therefore, the score normalizer component was trained on system output on half of the dev set.  As a result the full dev set results are contaminated.  The fair evaluation of the submission is on the leaderboard itself.

```

Add link to leaderboard here?

File: notebooks/boolqa/eval_predictions.ipynb [hunk 4]
```diff
@@ -359,12 +359,12 @@
...
   {
    "cell_type": "markdown",
-   "id": "6c49dbb3",
+   "id": "3a9dfa92",
    "metadata": {},
    "source": [
     "# Samples of EVC output \n",
-    "As above this classifier adds two new fields.  `boolean_answer_pred` is `yes` if the predicted answer to a boolean question is positive/true/yes, `no` if the answer is negative/false/no, and `no_answer` if there is no support for either answer in the context.  The field `boolean_answer_scores` provides the scores (logits) of each class.\n",
-    "For the TydiQA evaluation, we discard the `no_answer` prediction and always predict `yes` or `no`.  Other application may choose a different behavior.\n",
+    "As above this classifier adds two new fields.  `boolean_answer_pred` is `yes` if the predicted answer to a boolean question is positive/true/yes, `no` if the answer is negative/false/no.   The field `boolean_answer_scores` provides the scores (logits) of each class.\n",
```
Comment (seirasto) [inline notebooks/boolqa/eval_predictions.ipynb:366]:
```suggestion
    "As shown above this classifier adds two new fields: 

1.  `boolean_answer_pred` is `yes` if the predicted answer to a boolean question is positive/true/yes, and `no` if the answer is negative/false/no.
2.  `boolean_answer_scores` provides the scores (logits) of each class.\n",

```

File: primeqa/mrc/README.md [hunk 5]
```diff
@@ -260,9 +260,9 @@ Users can also train (fine-tune) and evaluate the MRC model on custom data by pr
 PrimeQA also supports special features for MRC systems as follows:
 
 ### Boolean Questions
-Answering [Boolean Questions](https://arxiv.org/abs/1905.10044) for TyDI (currently in an inference-only setup). Please read the [details](https://primeqa.github.io/primeqa/api/boolqa/index.html)):
+Answering [Boolean Questions](https://arxiv.org/abs/1905.10044) for TyDI. Please read the [details](https://primeqa.github.io/primeqa/api/boolqa/index.html)) of inference, or about [training](https://primeqa.github.io/primeqa/examples/boolqa/index.html):
```
Comment (seirasto) [inline primeqa/mrc/README.md:263]:
```suggestion
Answering [Boolean Questions](https://arxiv.org/abs/1905.10044) for TyDI. Please read the details of [inference](https://primeqa.github.io/primeqa/api/boolqa/index.html) or [training](https://primeqa.github.io/primeqa/examples/boolqa/index.html):
```

**Review by seirasto — APPROVED** [review]
LGTM. I've left a couple of comments regarding typos. Also please see comment regarding the model names having "tydi-tydi"

**Review by rongzhangibm — APPROVED** [review]
LGTM

**Review by franzmpub — APPROVED** [review]
LGTM
